capi/Makefile.am

```diff
@@ -17,6 +17,7 @@ libyami_capi_ldflags = \
 
 libyami_capi_cppflags = \
 	$(LIBVA_CFLAGS) \
+	-I$(top_srcdir)/interface \
```

Comment:
Keep in mind the users of libyami.  Clearly on the Capi header needs change the rest should be included inside libyami in the same way as the other elements of the library as already mentioned.

I know there are different ways to resolve this problem and we choose not to break users first.